pirate-radio.lua

```diff
@@ -123,6 +123,10 @@ function redraw_timer_init()
       menu_activated = true
     end
     frame_counter = frame_counter+1
+    local playback_rate = params:get("playback_rate")
```

Comment:
seems like we launch the redraw metro before params get set, so had to make lookup conditional